spec/aws_support/matchers/create_an_aws_object.rb

```diff
@@ -8,17 +8,20 @@ class CreateAnAWSObject
       include RSpec::Matchers::Composable
       include AWSSupport::DeepMatcher
 
-      def initialize(example, resource_class, name, expected_values)
+      def initialize(example, resource_class, name, expected_values, custom_matcher)
```

Comment:
Maybe a comment noting that `custom_matcher` is a block? One less thing to track down, for anyone spelunking in the code.